### PR TITLE
FIX-BESTPRACTICES: vm-custom-script-windows

### PR DESCRIPTION
**Pull request by badri-spektra**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

File: quickstarts/microsoft.compute/vm-custom-script-windows/azuredeploy.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
       "metadata": {
         "description": "The base URI where artifacts required by this template are located. When the template is deployed using the accompanying scripts, a private location in the subscription will be used and this value will be automatically generated."
       },
-      "defaultValue": "[deployment().properties.templateLink.uri]"
```
Comment (bmoore-msft) [inline quickstarts/microsoft.compute/vm-custom-script-windows/azuredeploy.json:22]: revert this change

File: quickstarts/microsoft.compute/vm-custom-script-windows/azuredeploy.json [hunk 2]
```diff
@@ -46,15 +53,15 @@
     "virtualNetworkName": "MyVNET",
     "publicIPAddressName": "myPublicIP",
     "dnsNameForPublicIP": "[uniqueString(resourceGroup().id)]",
-    "scriptFolder": ".",
+    "scriptFolder": "Scripts",
     "scriptFileName": "Copy-FileFromAzure.ps1",
```
Comment (bmoore-msft) [inline quickstarts/microsoft.compute/vm-custom-script-windows/azuredeploy.json:57]: revert this change and move the files back to the root

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

**Comment by badri-spektra** [comment]
@bmoore-msft there are two templates with names "list-storage-keys-windows-vm" and "vm-custom-script-windows", if we update artifacts location of these two templates main template is getting failed. these two templates are working only if we use absolute path in artifacts location

PR numbers 11664 & 11694

**Comment by bmoore-msft** [comment]
fixed in a later PR

**Pull request closed without merging**